src/symmbasis.jl

```diff
@@ -49,20 +49,17 @@ function SymmetricBasis(pibasis, φ::TP) where {TP}
    # TODO: should this be stored with the basis?
    #       or maybe written to a file on disk? and then flushed every time
    #       we finish with a basis construction???
-   rotc = Rot3DCoeffs(rfltype(pibasis))
-
+   #rotc = Rot3DCoeffs(rfltype(pibasis))
+   rotc = Rot3DCoeffsEquiv(φ, rfltype(pibasis))
    # allocate triplet format
    Irow, Jcol, vals = Int[], Int[], TP[]
    # count the number of PI basis functions = number of rows
    idxB = 0
 
    # loop through AA basis, but skip most of them ...
    for (iAA, AA) in enumerate(AAspec)
-      # AA = [b1, b2, ...], each bi = (n = ..., l = .., m = ...)
```

Comment:
why were these comments deleted? anything incorrect about this?

Reply:
or was this me?

Reply:
Same as below. I think I must have accidentally deleted these when I removed some of my own comments